src/Compilers/Server/VBCSCompilerTests/DesktopBuildClientTests.cs

```diff
@@ -22,53 +22,6 @@ namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
     public abstract class DesktopBuildClientTests : TestBase
```

Comment:
should this file and class be renamed?

Reply:
Will be in a future PR. Didn't want to rename + change all the content cause it could potentially make the review harder. 

---
In reply to: [319659438](https://github.com/dotnet/roslyn/pull/38411#discussion_r319659438) [](ancestors = 319659438)